src/components/SearchTracePage/TraceSearchForm.js

```diff
@@ -229,7 +238,15 @@ export function TraceSearchFormImpl(props) {
 
         {selectedLookback === 'custom' && (
           <div className="search-form--end-time field js-test-end-input">
-            <label htmlFor="service">End time</label>
+            <label htmlFor="service">
+              End time{' '}
+              <Popup
+                on="click"
+                wide="very"
+                trigger={<i className="SearchForm--hintTrigger info circle icon grey" />}
+                content={<h5>Times are expressed in {tz} timezone and converted to UTC</h5>}
```

Comment:
I would remove the part "and converted to UTC" from the tool tip, I don't think it's relevant to the user experience.